tasks/configurator.js

```diff
@@ -13,15 +15,96 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 const gulp = require('gulp-help')(require('gulp'));
 const config = require('./config');
 const util = require('gulp-util');
+const runSequence = require('run-sequence');
 const postcss = require('gulp-postcss');
 const replace = require('gulp-replace');
 const csstree = require('css-tree');
 const rename = require('gulp-rename');
 const through = require('through2');
+const webpack = require("webpack");
+const webpackProdConfig = require('../webpack.config.js')({
+  prod: true
+});
+const webpackDevConfig = require('../webpack.config.js')({
+  dev: true
+});
+const webpackTestConfig = require('../webpack.config.js')({
+  test: true
+});
+const WebpackDevServer = require("webpack-dev-server");
+const KarmaServer = require('karma').Server;
+
+function configuratorWebpackProd() {
+  webpack(webpackProdConfig, function(err, stats) {
+    if (err) {
+      throw new util.PluginError("webpack", err);
+    }
+
+    // Output the webpack stats
+    util.log("[webpack]", stats.toString({}));
+  });
+}
+
+function runWebpackDevServer(webpackConfig) {
+  if(webpackConfig.entry && Array.isArray(webpackConfig.entry)) {
+    webpackConfig.entry.unshift("webpack-dev-server/client?http://localhost:8080");
+  }
+  const webpackCompiler = webpack(webpackConfig);
+  new WebpackDevServer(webpackCompiler, {
+        publicPath: "/"
+    }).listen(8080, "localhost", function(err) {
+        if (err) {
+          throw new util.PluginError("webpack-dev-server", err);
+        }
+
+        // Inform the user of the url they can reach the server
+        util.log("[webpack-dev-server]", "Development Server: http://localhost:8080");
+        util.log("[webpack-dev-server]", "Directory Listing for current server: http://localhost:8080/webpack-dev-server");
+    });
+}
+
+function configuratorServe() {
+  runWebpackDevServer(webpackDevConfig);
+}
+
+function configuratorWatchProd() {
+  return gulp.watch(
+      [
+        `${config.src.configurator}/**/*`
+      ],
+      function(event) {
+        runSequence('configurator:webpack');
+      });
+}
+
+function runKarma(options, callback) {
+
+  // Parse our karma config
+  karmaServerConfig = {
+    configFile: `${__dirname}/../karma.conf.js`
+  };
+
+  // Pass in our options
+  Object.keys(options).forEach(function(key) {
```

Comment:
Hahahahaha! You are right, that is what I get for copying code from the webpack docs. Thanks!